longruntests/longruntests_suite_test.go

```diff
@@ -0,0 +1,152 @@
+package longruntests_test
+
+import (
+	"fmt"
+	"io/ioutil"
+	"math/rand"
+	"os"
+	"os/exec"
+	"path"
+	"runtime"
+	"testing"
+	"time"
+
+	. "github.com/deis/workflow-e2e/utils"
```

Comment:
I'd rather not anonymous import this package. The below ones are fine, as we've settled on that convention in this repo. Let's not expand it further, though.